tests/tests/swfs/avm2/sprite_with_frames/test.toml

```diff
@@ -0,0 +1,7 @@
+num_frames = 2
+
+[image_comparison]
+tolerance = 0
+
+[player_options]
+with_renderer = { optional = true, sample_count = 1 }
```

Comment:
This should be optional = false, unless you add some traces to show when it succeeds/fails